configure.in

```diff
@@ -2713,6 +2708,9 @@ if test "$wxUSE_LIBTIFF" != "no" ; then
             dnl we need to hack around this
             ac_configure_args="$ac_configure_args --disable-jpeg"
         fi
+        dnl We don't provide a built-in version of the libjbig library, so
```

Comment:
Sorry to be such a pedant, but we seem to have lost ` if test "$wxUSE_LIBJBIG" != "yes"; then` condition around these lines. I'm not sure if anybody uses this option, but as long as it exists, it would seem to be better to keep it working as before, i.e. not disable jbig support in libtiff it it's specified.

Reply:
jbig support is supposed to be disabled when using the builtin libtiff because it isn't supplied. This was added into line 551 (https://github.com/wxWidgets/wxWidgets/blob/master/configure.in#L551), but that tests for `wxUSE_LIBTIFF` being builtin at the beginning. What this change does is make it so jbig support is always disabled no matter the path leading to the builtin library configure (since before it also wouldn't disable the jbig support when falling back from the system library).

Reply:
Thanks, I've missed the check at the line 551, with it (and your explanations) I can indeed see that this is correct, sorry for the false alarm.